common/v2/translations/lang/en.json

```diff
@@ -269,6 +269,16 @@
 		"EMAIL_ADDRESS_PLACEHOLDER": "Email address",
 		"ENCLAVE_LEDGER_FAIL": "Failed to connect to Ledger",
 		"ENCLAVE_LEDGER_IN_USE": "Your Ledger is currently in use with another application. Please wait, or close other wallet applications before trying again.",
+		"ENS_DASHBOARD_HEADER": "ENS Dashboard",
+		"ENS_MY_DOMAINS_TABLE_HEADER": "Your ENS Names",
+		"ENS_MY_DOMAINS_TABLE_OWNER_ADDRESS_HEADER": "Owner Address",
+		"ENS_MY_DOMAINS_TABLE_DOMAIN_NAME_HEADER": "Domain Name",
+		"ENS_MY_DOMAINS_TABLE_EXPIRES_HEADER": "Expires",
+		"ENS_DOMAIN_UNKNOWN_NAME": "Unknown ENS Name",
+		"ENS_EXPIRING_SOON": "This ENS name is expiring within 30 days",
+		"ENS_DOMAINS_NO_DOMAINS": "No ENS domains detected",
+		"ENS_DOMAINS_NO_DOMAINS_MORE": "When at least one of your accounts owns an ENS domain, it will display here. [Click here to get one!](https://app.ens.domains)",
```

Comment:
I think we're moving away from having constant URLs in the language file?

Reply:
Are we? There seems to be a bunch of them still in there :3.

Reply:
@FrederikBolding It spoke too soon about the i18n and it seems we still rely on the keys.